itest/tests/exportResults.test.js

```diff
@@ -0,0 +1,57 @@
+/* @flow */
+import fsExtra from "fs-extra"
+
+import dialog from "spectron-fake-dialog"
+import path from "path"
+
+import {
+  click,
+  newAppInstance,
+  pcapIngestSample,
+  startApp,
+  startSearch,
+  writeSearch
+} from "../lib/app"
+import {locator} from "../../src/js/test/integration"
+import {retryUntil} from "../lib/control"
+
+describe("test exporting results", () => {
+  let app
+  let testIdx = 0
+
+  beforeAll(async () => {
+    app = newAppInstance(path.basename(__filename), ++testIdx)
+    dialog.apply(app)
+    await startApp(app)
+    await pcapIngestSample(app)
+    await fsExtra.ensureDir("tmp")
+  })
+
+  afterAll(async () => {
+    if (app && app.isRunning()) await app.stop()
+    await fsExtra.remove("tmp")
+  })
+
+  test("clicking the button", async () => {
```

Comment:
I missed this before. This should be a `stdTest` so that we get logging when the test starts, and it should call the standard handler if there's an exception.